src/transformers/models/siglip/modeling_siglip.py

```diff
@@ -488,7 +500,7 @@ def _init_weights(self, module):
             nn.init.xavier_uniform_(module.attention.in_proj_weight.data)
             nn.init.zeros_(module.attention.in_proj_bias.data)
         elif isinstance(module, SiglipModel):
-            logit_scale_init = torch.log(torch.tensor(1.0))
+            logit_scale_init = torch.tensor(0.0)
```

Comment:
`torch.log` does not work on CPU + fp16, but also, why write 2 as 1 + 1 when you can write 2 directly?